agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/DatabaseClientDecorator.java

```diff
@@ -18,26 +18,27 @@
 
 import io.opentelemetry.trace.Span;
 import io.opentelemetry.trace.attributes.SemanticAttributes;
+import io.opentelemetry.trace.attributes.StringAttributeSetter;
 
 /** @deprecated use {@link DatabaseClientTracer} instead. */
 @Deprecated
 public abstract class DatabaseClientDecorator<CONNECTION> extends ClientDecorator {
 
-  protected abstract String dbType();
+  protected abstract String dbSystem();
 
   protected abstract String dbUser(CONNECTION connection);
 
-  protected abstract String dbInstance(CONNECTION connection);
+  protected abstract String dbName(CONNECTION connection);
 
   // TODO make abstract after implementing in all subclasses
-  protected String dbUrl(final CONNECTION connection) {
+  protected String dbConnectionString(final CONNECTION connection) {
     return null;
   }
 
   @Override
   public Span afterStart(final Span span) {
     assert span != null;
-    span.setAttribute(SemanticAttributes.DB_TYPE.key(), dbType());
+    span.setAttribute(StringAttributeSetter.create("db.system").key(), dbSystem());
```

Comment:
This seems to be used in a lot of places.  Perhaps it should be created once somewhere and reused?

Reply:
oh, I see... that's a subject for a follow on PR.

Reply:
Yup, it will be replaced with SemanticAttributes.DB_SYSTEM.